modules/test/playwright/tests/frontend-data-set-views-web/pages/ActionsPage.ts

```diff
@@ -49,9 +49,9 @@ export class ActionsPage {
 
 	async goto() {
 		await this.viewsPage.goto();
-		await this.viewsPage.gotoSampleDataSetView();
-
-		await this.page.getByRole('button', {name: 'Actions'}).first().click();
+		await this.viewsPage.gotoSampleDataSetView().then(() => {
+			this.page.getByRole('button', {name: 'Actions'}).first().click();
```

Comment:
:thinking: Does this mean that playwright does not "wait" after last click in `gotoSampleDataSetView()`?

Should we consider chaining all the gotos as the way of composing a predictable navigation?